### PR TITLE
[CMDCT-3366] - Unskip and split up accessibility tests

### PR DESCRIPTION
**Pull request by ntsummers1**
### Description

Accessibility tests should now be running for MFP! 

### Related ticket(s)

CMDCT-3366

---
### How to test

Note the password is the same for logging in as a state user or admin user in MFP. If you don't have the password or think its been updated, check the proj-cms-mdct-devs channel for the pinned message at the top called MDCT App Access
- Navigate to tests/cypress
- Run `CYPRESS_ADMIN_USER_PASSWORD='insertpasswordhere!' CYPRESS_STATE_USER_PASSWORD='Insertpasswordhere!' yarn test`
- Click e2e configuration
- Run the new tests! 

### Important updates

Added a slick testPageAccessibility command to reduce the amount of copied code. Can use for any page!

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

### CODE REVIEW

**Review by jessabean — APPROVED** [review]

**Comment by keeysnc** [comment]
Hey @ntsummers1 just for clarification, these tests were already set up? They just needed a few bug fixes?

**Review by keeysnc — COMMENTED** [review]
The login spect worked perfectly but the homepage spec didnt run for me. I'm not sure if it was something I had to do on my end though.

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -36,6 +42,14 @@ Cypress.Commands.add("navigateToHomePage", () => {
   if (cy.location("pathname") !== "/") cy.visit("/");
 });
 
+Cypress.Commands.add("testPageAccessibility", () => {
+  Object.keys(breakpoints).forEach((deviceSize) => {
+    const size = breakpoints[deviceSize];
+    cy.viewport(...size);
+    cy.runAccessibilityTests();
+  });
```
Comment (benmartin-coforma) [inline tests/cypress/support/commands.js:50]: Suggestion: Use Object.values to skip the object lookup, and `for...of` over `.forEach`

```suggestion
  for (let size of Object.values(breakpoints)) {
    cy.viewport(...size);
    cy.runAccessibilityTests();
  }
```

**Review by benmartin-coforma — COMMENTED** [review]
Excellent work; I love the new cypress command.

**Review by britt-mo — DISMISSED** [review]
Ran locally, all tests passed

**Review by jessabean — APPROVED** [review]

**Review by benmartin-coforma — APPROVED** [review]
Thank you!

**Pull request merged**